apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClientImpl.Queries.cs

```diff
@@ -167,33 +125,41 @@ public override BigQueryResults GetQueryResults(JobReference jobReference, GetQu
         }
 
         /// <inheritdoc />
-        public override async Task<BigQueryJob> CreateQueryJobAsync(BigQueryCommand command, CreateQueryJobOptions options = null, CancellationToken cancellationToken = default(CancellationToken))
+        public override async Task<BigQueryJob> CreateQueryJobAsync(BigQueryCommand command, QueryOptions options = null, CancellationToken cancellationToken = default(CancellationToken))
         {
             GaxPreconditions.CheckNotNull(command, nameof(command));
             var query = new JobConfigurationQuery { UseLegacySql = false };
             command.PopulateJobConfigurationQuery(query);
             var job = await CreateInsertQueryJobRequest(query, options).ExecuteAsync(cancellationToken).ConfigureAwait(false);
             return new BigQueryJob(this, job);
-        }
+        }        
```

Comment:
Thanks. I've clearly got something adding this in the way that I do things - it happens far too often :(